pkgs/development/tools/headache/default.nix

```
@@ -1,22 +1,24 @@
-{ lib, fetchFromGitHub, ocamlPackages }:
+{ lib, fetchFromGitHub, nix-update-script, ocamlPackages }:
 
 with ocamlPackages;
 
 buildDunePackage rec {
   pname = "headache";
-  version = "1.06";
+  version = "1.07";
 
   src = fetchFromGitHub {
     owner = "frama-c";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-BA7u09MKYMyspFX8AcAkDVA6UUG5DKAdbIDdt+b3Fc4=";
+    sha256 = "sha256-RL80ggcJSJFu2UTECUNP6KufRhR8ZnG7sQeYzhrw37g=";
   };
 
   propagatedBuildInputs = [
     (camomile.override { version = "1.0.2"; })
```

```suggestion
    camomile
```
Headache has been updated to camomile 2.0 according to release notes, and it doesn't build with current setup

---

Indeed! I think it's what made me start #242103 months ago!